connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -237,7 +213,7 @@ export async function deleteTicketBatchActivity(
 }
 
 /**
- * Retrieves the timestamp cursor, which is the start date of the last successful sync.
+ * Retrieves the timestamp cursor, which is the start date of the last successful incremental sync.
```

Comment:
so what is the difference with a full sync here?
full sync = incremental sync with cursor at 0? (if yes no need to indicate the 'incremental')
when setting up the connector, we do a full sync. There has not been incremental sync yet. What happens for the first incremental sync?

Reply:
I think I missed something and need to initialize the cursor at the first sync that is not incremental.
On the first incremental sync, the current behavior is to start at `Date.now()` - 5 min (cron schedule every 5 min), which should be the start of the previous sync.

Reply:
To clarify the points you raised:
- By full-sync I refer to syncs that are not incremental (the term is a bit incorrect).
- Full syncs use other endpoints than the incremental syncs and retrieve all the data in a straightforward manner.
- Brands are the top-level nodes of the content tree, on incremental syncs we loop over the brands, since the incremental sync is common to all brands we can keep a single cursor.